### PR TITLE
Openable windows without curtains

### LINKED ISSUE #8165: Tearing down curtains prevents opening window
build : f64e68e
OS : Win7
Tielset : Yes

Close the curtain and press e to interact with curtain.
Choose tear down the curtains
After you have done that you are no longer able to open the window.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/2996080-tearing-down-curtains-prevents-opening-window?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### LINKED ISSUE #13858: Closed windows
After you tear down a cutrain from a window in a house, you can't open it.

### PR DESCRIPTION
**Pull request by OzoneH3**
It's not pretty, but it gets the job done.

Closes #13858
Closes #7749
Fixes #8165

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -461,7 +461,7 @@
             [ [ "glass_sheet", 1 ] ]
         ],
         "pre_terrain" : "t_window_empty",
-        "post_terrain" : "t_window"
+        "post_terrain" : "t_window_no_curtains"
     },{
         "type" : "construction",
         "description" : "Build Window", "//": "Step 3: window with curtains",
```
Comment (OzoneH3) [inline data/json/construction.json]: Not sure if this one should even stay, you won't be able to build t_window just find them in stores.

Comment (Zireael07) [inline data/json/construction.json]: Nice solution to the store window being special!

**Comment by BevapDin** [comment]
There are a few mapgens of houses that should probably use the new windows.

data/json/mapgen/house_quiverfull.json comes to mind, it currently has only standard "t_window" (without curtains), which looks a bit strange for a private building.

**Comment by OzoneH3** [comment]
I can fix those houses if someone points me to them but otherwise "fixing" houses is not really in the scope of this pr.

As for house_quiverfull should it be t_window_domestic or was it intended for the windows to have no curtains and use t_window_no_curtains?

File: data/json/construction.json [hunk 2]
```diff
@@ -481,6 +481,23 @@
         "post_terrain" : "t_window_domestic"
     },{
         "type" : "construction",
+        "description" : "Build Window", "//": "Step 3: window with curtains",
+        "category" : "CONSTRUCT",
+        "difficulty" : 2,
+        "time" : 30,
+        "qualities": [ [
+            { "id": "SAW_W", "level": 1 }
```
Comment (Coolthulhu) [inline data/json/construction.json]: Should probably have HAMMERING 2 too, to drive the nails in.

**Comment by Coolthulhu** [comment]
This new window needs some different name and preferably either color or glyph too.
Currently it's not possible to tell which window has curtains without walking up to it first.

**Comment by BevapDin** [comment]
> As for house_quiverfull should it be t_window_domestic or was it intended for the windows to have no curtains and use t_window_no_curtains?

I though it was intended to be windows without curtains. A private house that has windows that can not be opened looks strange to me, but that may be intended. @jokermatt999 (the creator) should know.

**Comment by jokermatt999** [comment]
Nope, just a mistake. I've been meaning to get back into developing, so
I'll whip up a fix later tonight. I've got a few buildings I wanted to
tweak actually.

**Comment by OzoneH3** [comment]
So, I'll leave the house_quiverfull to jokermatt999 then.

Recolored domestic windows:
w/ curtains: c_ltgrey
w/o curtains: c_white
Left the commercial ones ltcyan and all taped variants dkgrey.

**Comment by MisterFelixFox** [comment]
I never thought they'd get used!

**Pull request merged**